modules/get_optimizer.py

```diff
@@ -0,0 +1,8 @@
+from torch.optim import SGD, Adam
+
+def get(model, optimizer_type, learning_rate): #
```

Comment:
Same comments as for the get_scheduler
```suggestion
def get(model, optimizer_type, learning_rate):
```